EssentialFeed/EssentialFeedTests/Image Comments/Presentation/ImageCommentsPresenterTests.swift

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+
+class ImageCommentsPresenterTests: XCTestCase {
+	func test_title_isLocalized() {
+		XCTAssertEqual(ImageCommentsPresenter.title, localized("IMAGE_COMMENTS_VIEW_TITLE"))
+	}
+
+	func test_map_createsViewModel() {
+		let now = Date()
+		let calendar = Calendar(identifier: .gregorian)
+
+		let comments = [
+			ImageComment(
+				id: UUID(),
+				message: "a message",
+				createdAt: now.adding(minutes: -5, calendar: calendar),
+				username: "a username"),
+			ImageComment(
+				id: UUID(),
+				message: "another message",
+				createdAt: now.adding(days: -1, calendar: calendar),
+				username: "another username")
+		]
+
+		let viewModel = ImageCommentsPresenter.map(comments, currentDate: now, calendar: calendar, locale: Locale(identifier: "en_US_POISX"))
```

Comment:
typo (`en_US_POISX` should be `en_US_POSIX`):

```suggestion
		let viewModel = ImageCommentsPresenter.map(comments, currentDate: now, calendar: calendar, locale: Locale(identifier: "en_US_POSIX"))
```